config.yaml

```diff
@@ -1,18 +1,32 @@
 options:
-  prometheus-image-path:
+  image-tags-url:
     type: string
     description: |
-      The location of the image to use,
-      e.g. "registry.example.com/prometheus:v1".
-
-      This setting is required.
-    default: "prom/prometheus:latest"
+      URL at which all the tags for the Prometheus container images can
+      be viewed and verified.
+    default: "https://registry.hub.docker.com/v1/repositories/prom/prometheus/tags"
+  image-registry-path:
+    type: string
+    description: |
+      Path from which to pull the Prometheus container image.
+    default: "prom/prometheus"
+  prometheus-image-major-version:
+    type: string
+    description: |
+      The major version of the Prometheus image to use.
+      Patch versions will automatically be set to the latest version.
+    default: "2"
+  prometheus-image-minor-version:
+    type: string
+    description: |
+      The minor version of the Prometheus image to use.
+      Patch versions will automatically be set to the latest version.
+    default: "20"
   prometheus-image-username:
```

Comment:
Similar to image-path and registry-path, the image we need is not protected, and will not need a username/password. Let's remove these configs now, to prevent future breaking changes.